src/components/app/App.js

```diff
@@ -5,257 +5,27 @@ import './App.css';
 
 export default class App extends Component {
 
-render() {
-    const users = [
```

Comment:
No need to delete it yet. Because we will use it in Reducer to test that everything still works.